src/docs/cookbook/navigation/navigate-with-arguments.md

```diff
@@ -10,25 +10,25 @@ next:
  
 The [`Navigator`]({{site.api}}/flutter/widgets/Navigator-class.html)
 provides the ability to navigate to a named route from any part of an app using
-a common identifier. In some cases, you may also need to pass arguments to a
-named route. For example, you may wish to navigate to the `/user` route and
+a common identifier. In some cases, you might also need to pass arguments to a
+named route. For example, you might wish to navigate to the `/user` route and
 pass information about the user to that route.
 
-In Flutter, you can accomplish this task by providing additional `arguments` to
-the
-[`Navigator.pushNamed`]({{site.api}}/flutter/widgets/Navigator/pushNamed.html)
-method. You can extract the arguments using the
+In Flutter,
+you can accomplish this task by providing additional `arguments` to the
```

Comment:
`arguments` confused me. Maybe:

...task using the `arguments` parameter of the...